content/doc/best-practices/tips_and_tricks.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Tips & Tricks
+shortdesc: Discover some useful tips and tricks to improve your experience with Clever Cloud tools
+tags:
+- reference
+- tips
+- tricks
+keywords:
+- tips
+- tricks
+type: docs
+---
+
+Clever Cloud is an easy-to-use platform, designed with developers in mind. Whether you're a beginner or a more experienced user, we try to fit your needs bringing you the best possible experience through our web Console, Clever Tools, API, SDK, integrations.
+
+We like to pay attention to those small details, those features, that make Clever Cloud better to use, unique. Over the time, we've developed them but found out that most of our users or new team members don't know about them. This page exists to help spread the word.
+
+## Web Console
+
+Access Clever Cloud Console to manage your applications, add-ons, organizations and so on: https://console.clever-cloud.com
+
+### Menu advanced filter
+
+The left side menu is a powerful tool with a text field to filter your applications and add-ons. You can type the name of the items you want to find, and see results matching your search instantly. But do you know that you can also use specific keywords ? Here is the list:
+
+| Keyword               | Description |
+| -------               | ----------- |
+| `is:addon`            | Filter add-ons |
+| `is:app`              | Filter applications |
+| `is:[addon-provider]` | Filter by add-on providers (for example, `is:kv`, `is:keycloak`, `is:postgresql`) |
+| `is:[runtime]`        | Filter by runtime name (for example, `is:node`, `is:php`) |
+| `is:[variant]`        | Filter by variant name (for example, `is:static-apache`) |
+
+### Environment variables are multi-line
+
+When you edit an environment variable, in simple mode, you see a text input for each variable. But if you press the `Enter` key, the value splits into multiple lines. Thus, you can put the full content of a configuration file in it for example. Of course, you can do the same with expert and JSON modes, through Clever Tools or the API. Environment variables ARE multi-line.
```

Comment:
Also, when adding a new variable, `Ctrl+Enter` will validate the variable and put the cursor back to the now empty "Variable name" field.